src/types/Config.tsx

```diff
@@ -533,38 +549,38 @@ export const ALL_PAGES: IPage[] = [
   },
   {
     name: PAGES.COMPANY_ROLE,
-    role: ROLES.UPDATE_COMPANY_ROLE,
+    allowTo: () => userHasPortalRole(ROLES.UPDATE_COMPANY_ROLE),
     element: <CompanyRoleUpdate />,
   },
   {
     name: PAGES.USECASE_PARTICIPATION,
-    role: ROLES.REQUEST_SSICREDENTIAL,
+    allowTo: () => userHasPortalRole(ROLES.REQUEST_SSICREDENTIAL),
     element: <UsecaseParticipation />,
   },
   {
     name: PAGES.CERTIFICATE_CREDENTIAL,
-    role: ROLES.REQUEST_SSICREDENTIAL,
+    allowTo: () => userHasPortalRole(ROLES.REQUEST_SSICREDENTIAL),
     element: <CertificateCredentials />,
   },
   { name: PAGES.DATA_SPACE, element: <DataSpace /> },
   {
     name: PAGES.ADMIN_CREDENTIAL,
-    role: ROLES.DECISION_SSICREDENTIAL,
+    allowTo: () => userHasSsiCredentialRole(ROLES.DECISION_SSICREDENTIAL),
     element: <AdminCredential />,
   },
   {
     name: PAGES.ONBOARDING_SERVICEPROVIDER,
-    role: ROLES.IDP_VIEW,
+    allowTo: () => userHasPortalRole(ROLES.IDP_VIEW),
     element: <OSPManagement />,
   },
   {
     name: PAGES.COMPANY_CERTIFICATE,
-    role: ROLES.COMPANY_CERTIFICATE_VIEW,
+    allowTo: () => userHasPortalRole(ROLES.COMPANY_CERTIFICATE_VIEW),
     element: <CompanyCertificates />,
   },
   {
     name: PAGES.COMPANY_WALLET,
-    role: ROLES.CREDENTIAL_REQUESTS,
+    allowTo: () => userHasPortalRole(ROLES.CREDENTIAL_REQUESTS),
```

Comment:
@oyo update needed - access is
<img width="291" alt="image" src="https://github.com/eclipse-tractusx/portal-frontend/assets/94133633/b5e1e2c2-ac04-470d-8ddc-1432136f3162">